src/data/news/decred_journals.yml

```diff
@@ -1,75 +1,107 @@
+-
+  Title: "Decred Journal, July 2023"
+  Permalink: "https://www.cypherpunktimes.com/decred-journal-july-2023/"
+  Params:
+    author: "Decred Journal Team"
+    sortDate: "2023-07-22"
```

Comment:
```suggestion
    sortDate: "2023-08-22"
```

Reply:
Good catch! I re-checked the diff so many times but still missed this bad paste.